l2geth/internal/ethapi/backend.go

```diff
@@ -59,7 +59,7 @@ type Backend interface {
 	StateAndHeaderByNumberOrHash(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) (*state.StateDB, *types.Header, error)
 	GetReceipts(ctx context.Context, hash common.Hash) (types.Receipts, error)
 	GetTd(hash common.Hash) *big.Int
-	GetEVM(ctx context.Context, msg core.Message, state *state.StateDB, header *types.Header) (*vm.EVM, func() error, error)
+	GetEVM(ctx context.Context, msg core.Message, state *state.StateDB, header *types.Header, vmCfg vm.Config) (*vm.EVM, func() error, error)
```

Comment:
In upstream this is a pointer:

```
, vmCfg vm.Config
```

https://github.com/ethereum/go-ethereum/blob/64c53edf836efb1d55b85e841db364a424e95422/internal/ethapi/backend.go#L70